cpp/system_updater.cc

```diff
@@ -131,9 +131,17 @@ int Main(int argc, char *argv[]) {
 
     std::sort(script_names.begin(), script_names.end(), ScriptLessThan);
     std::string last_schema, current_schema;
+    int last_version_number = -1;
```

Comment:
`previous_version_number` should be a better variable name than `last_version_number`